nodeup/pkg/bootstrap/install.go

```diff
@@ -33,21 +32,17 @@ import (
 )
 
 type Installation struct {
-	FSRoot          string
 	CacheDir        string
 	RunTasksOptions fi.RunTasksOptions
 	Command         []string
 }
 
 func (i *Installation) Run() error {
-	distribution, err := distros.FindDistribution(i.FSRoot)
+	_, err := distros.FindDistribution("/")
```

Comment:
I think we could get rid of the string argument in `FindDistribution()` since it is only ever passed `"/"`

Reply:
Maybe we want to add some tests for it in the future, or use in some container?